hooks/useUser.ts

```diff
@@ -130,6 +145,12 @@ const useUser = () => {
       }
 
       const publicKey = bufferToBase64URLString(authenticatorReponse.response.getPublicKey())
+      const coordinates = await decodePublicKeyForWeb(publicKey)
```

Comment:
@LiorAgnin decodePublicKeyForWeb is fine for backend. However in frontend, decodePublicKey need to be used to handle both Web and ReactNative:
https://github.com/safe-global/safe-core-sdk/blob/0b3378478e88d9303d04f314a477228b0cbe221f/packages/protocol-kit/src/utils/passkeys/extractPasskeyData.ts#L146